neurons/miners/inference_miner/miner.py

```diff
@@ -132,13 +135,15 @@ async def _forward(
         init_time = time.time()
         timeout_threshold = synapse.timeout
 
-        token_streamer = partial(
-            _forward,
-            self,
-            synapse,
-            init_time,
-            timeout_threshold,
-        )
+        with Timer() as timer:
+            token_streamer = partial(
+                _forward,
+                self,
+                synapse,
+                init_time,
+                timeout_threshold,
+            )
+        logger.info(f"Time for complete response: {timer.elapsed_time}")
         return synapse.create_streaming_response(token_streamer)
```

Comment:
It will measure the time for function creation, is it what was originally intended to do?
If you want to measure function call eg `Time for complete response`:
```suggestion
        with Timer() as timer:
            token_streamer = partial(
                _forward,
                self,
                synapse,
                init_time,
                timeout_threshold,
            )
            logger.info(f"Time for complete response: {timer.elapsed_time}")
            return synapse.create_streaming_response(token_streamer)
```